extension-definition-specifications/incident-core/Incident Extension Suite.adoc

```diff
@@ -244,9 +244,7 @@ This can be used to supplement the created_by_ref in cases where external author
 <<<
 [[event]]
 === 2.2. Event
-
-[width="100%",cols="100%",stripes=odd]
-|===
+This new sdo extension *MUST* use [stixliteral]#extension-definition--4ca6de00-5b0d-45ef-a1dc-ea7279ea910e# as its extension ID.
```

Comment:
Was removing this table formatting intentional?

Reply:
No - unintentional :-(

Please don't merge this PR - it was for Jeff @dc3-tsd